common/app/model/content/Atoms.scala

```diff
@@ -53,6 +54,11 @@ final case class MediaAtom(
       "^0".r.replaceFirstIn(formattedDuration, "") //strip leading zero
     }
   }
+
+  def activeAsset: MediaAsset = activeVersion match {
+    case Some(version) => assets.find(_.version == version).getOrElse(assets.head)
+    case _ => assets.head
+  }
```

Comment:
Worth mentioning that this won't work for Hosted as they have multiple assets on the same version number (this returns one). Not a big issue atm, as hosted pages have their own template. Maybe add a comment?